zipline/pipeline/filters/filter.py

```diff
@@ -593,6 +595,10 @@ class MaximumFilter(Filter, StandardOutputs):
     window_length = 0
 
     def __new__(cls, factor, groupby, mask):
+        from zipline.pipeline.classifiers import Everything
+        if groupby is NotSpecified:
+            groupby = Everything()
```

Comment:
If this import is going to be lazy anyway, is there a reason not to put it inside the if block?

Reply:
Probably not. I can move this down.